### PR TITLE
DIV-6516: Display General order documents

### PR DESCRIPTION
**Pull request by Dizrupto**
# Description

Display General orders on DA Frontend

Fixes https://tools.hmcts.net/jira/browse/DIV-6516

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by madjava — APPROVED** [review]

File: test/unit/steps/petitioner/progressBar.test.js [hunk 1]
```diff
@@ -100,7 +106,9 @@ describe(modulePath, () => {
         'costsOrder',
         'dnAnswers',
         'decreeNisi',
-        'decreeAbsolute'
+        'decreeAbsolute',
+        'generalOrder',
+        'generalOrder'
```
Comment (jacmurphy) [inline test/unit/steps/petitioner/progressBar.test.js:111]: Is this duplication necessary ? 1/2

File: test/unit/steps/respondent/progressBar.test.js [hunk 2]
```diff
@@ -97,7 +103,9 @@ describe(modulePath, () => {
         'certificateOfEntitlement',
         'costsOrder',
         'decreeNisi',
-        'decreeAbsolute'
+        'decreeAbsolute',
+        'generalOrder',
+        'generalOrder'
```
Comment (jacmurphy) [inline test/unit/steps/respondent/progressBar.test.js:108]: Is this duplication necessary ? 2/2

File: test/unit/steps/petitioner/progressBar.test.js (hunk 1 above)
Comment (jacmurphy) [reply test/unit/steps/petitioner/progressBar.test.js:111]: Yes, its in regards to the test data above that puts in two files.

File: test/unit/steps/respondent/progressBar.test.js (hunk 2 above)
Comment (jacmurphy) [reply test/unit/steps/respondent/progressBar.test.js:108]: Yes, its in regards to the test data above that puts in two files.

**Review by yarpo — APPROVED** [review]

**Pull request merged**